labgrid/resource/udev.py

```diff
@@ -224,6 +224,13 @@ def read_attr(self, attribute):
 @target_factory.reg_resource
 @attr.s(eq=False)
 class USBSerialPort(USBResource, SerialPort):
+    """This resource describes a USB serial port.
+
+    Args:
+        speed (int): baudrate
+    """
+    speed = attr.ib(default=115200, validator=attr.validators.instance_of(int))
+
```

Comment:
This shouldn't be required, the `SerialPort` class already contains the speed attribute.

Reply:
You are right. I fixed this and pushed again (I know, github workflow is about adding fix patches on top of the pull request. sorry for that).

Reply:
IMO it's fine to just force-push, since github now includes a diff between the force-pushes.